test/functional/wallet_avoidreuse.py

```diff
@@ -288,5 +290,36 @@ def test_getbalances_used(self):
         assert_unspent(self.nodes[1], total_count=2, total_sum=6, reused_count=1, reused_sum=1)
         assert_balances(self.nodes[1], mine={"used": 1, "trusted": 5})
 
+    def test_large_destination_group(self):
```

Comment:
Suggestion for line 202 if you have to retouch this commit:
```diff
-        self.log.info("Test fund send fund send")
+        self.log.info("Test fund send fund send {}".format(second_addr_type))
```

Reply:
Improved the naming overall in a separate commit. Thanks!